### ISSUE TITLE
Atari 2600

### ISSUE THREAD

**Issue by drrak** [libretro/prosystem-libretro#22]
_Google translator:_
The original Atari 7800 was compatible with games for the Atari 2600. It is a chance for the same compatibility in the emulator?

**Comment by underball** [comment]
No. The original version of this emulator (Prosystem) does not include 2600 comaptibility. If you want to play 2600 games, you need to use the Stella (2600) core in Retroarch.

**Comment by drrak** [comment]
Thanks.
